test/Validate.SSA.cpp

```diff
@@ -191,7 +199,7 @@ TEST_F(Validate, ForwardDecorateInvalidIDBad) {
 
 TEST_F(Validate, ForwardMemberDecorateGood) {
   char str[] = R"(
-           OpCapability Matrix
+           OpCapability Shader
```

Comment:
Indeed!   Every module needs an OpMemoryModel instruction, and  Logical requires Shader capability.

Interesting. Good.